G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookSaveAttachmentCommand.cs

```diff
@@ -13,15 +13,15 @@
 
 namespace G1ANT.Addon.MSOffice
 {
-    [Command(Name = "outlook.saveattachment", Tooltip = "This command saves attachment as a local file.")]
+    [Command(Name = "outlook.saveattachment", Tooltip = "This command saves an attachment to a file")]
     public class OutlookSaveAttachmentCommand : Command
     {
         public class Arguments : CommandArguments
         {
-            [Argument(Required = true)]
+            [Argument(Name = "Attachment", Tooltip = "Email attachment to be saved", Required = true)]
```

Comment:
it is correct of course but we never add `Name = ""` in the Argument attribute, the name of an argument is below so it's not needed

Reply:
```suggestion
            [Argument(Tooltip = "Email attachment to be saved", Required = true)]
```